modules/ztp-restarting-policies-reconciliation.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-restarting-policies-reconciliation_{context}"]
+= Restarting policies reconciliation
+
+Use the following procedure to restart policies reconciliation in the event of unexpected
+compliance issues.
+
+.Procedure
+
+. A `ClusterGroupUpgrade` custom resource (CR) is generated in the namespace `ztp-install`
+by the Topology Aware Lifecycle Operator (TALO) after the managed spoke cluster becomes `Ready`:
++
+[source,terminal]
+----
+$ export CLUSTER=<clusterName>
+----
++
+[source,terminal]
+----
+$ oc get clustergroupupgrades -n ztp-install $CLUSTER
+----
+
+. If there are unexpected issues and the policies fail to become complaint within the configured
+timeout, (the default is 4h), the status of the `ClusterGroupUpgrade` CR shows `UpgradeTimedOut`:
++
+[source,terminal]
+----
+$ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
+----
+
+. A `ClusterGroupUpgrade` CR in the `UpgradeTimedOut` state automatically restarts its policy
+reconciliation every hour. If you have changed your policies, you can start a retry immediately
+by deleting the existing `ClusterGroupUpgrade` CR. This triggers the automatic creation of a
+new `ClusterGroupUpgrade` CR that begins reconciling the policies immediately:
++
+[source,terminal]
+----
+$ oc delete clustergroupupgrades -n ztp-install $CLUSTER
+----
+
+Note that when the `ClusterGroupUpgrade` CR completes with status `UpgradeCompleted` and the
+managed spoke cluster has the label `ztp-done` applied, you can make additional configuration
+changes using `PolicyGenTemplate`. Deleting the existing `ClusterGroupUpgrade` CR will not
+make TALO generate a new CR.
+
+At this point, ZTP has completed its interaction with the cluster
+and any further interactions should be treated as an upgrade. See the Topology-Aware Lifecycle
+Operator (TALO) documentation for instructions on how to construct your own `ClusterGroupUpgrade` CR to
```

Comment:
Change this link to point to Alexandra's topic for TALO. PR: https://github.com/openshift/openshift-docs/pull/40392